integration/pkcs11/pkcs11_test.go

```diff
@@ -125,15 +142,18 @@ func configurePeerPKCS11(ctx *pkcs11.Ctx, sess pkcs11.SessionHandle, network *nw
 		Expect(err).NotTo(HaveOccurred())
 
 		By("Updating the peer signcerts")
-
 		newOrdererPemCert := buildCert(caBytes, orgCAPath, peerCSR, peerSerial, peerPubKey)
 		updateMSPFolder(network.PeerLocalMSPDir(peer), fmt.Sprintf("peer.%s-cert.pem", domain), newOrdererPemCert)
 
 		By("Updating the peer admin user signcerts")
 		newAdminPemCert := buildCert(caBytes, orgCAPath, adminCSR, adminSerial, adminPubKey)
-
 		orgAdminMSPPath := network.PeerUserMSPDir(peer, "Admin")
 		updateMSPFolder(orgAdminMSPPath, fmt.Sprintf("Admin@%s-cert.pem", domain), newAdminPemCert)
+
+		By("Updating the peer user1 signcerts")
```

Comment:
Let me see if I understand why you added this step: by updating the _user_ cert (that is, the one the CLI will use to sign proposal and transaction), are you ensuring that PKCS #11 is also used by the CLI? Assuming I got this right, would you agree that this would never happen in production since proposal/tx signing will be performed by the SDK which we don't exercise here?